lab_2_retrieval_w_bm25/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "target_score": 0
+    "target_score": 8
```

Comment:
Actually, I do not have comments. What about 10?

Reply:
done